src/Compiler/CompilerOptions.cpp

```diff
@@ -277,14 +279,38 @@ std::string getOptimizationLevelOption() {
 }
 
 // Support for Xopt.
-void setXoptOption(const std::string &flag) { Xopt = flag; }
+void setXoptOption(const std::vector<std::string> &flags) {
```

Comment:
@etiotto I wonder if you don't need to reset the list when setting the option. Because using the OnnxMlirCompiler interface, you can have a long running process that need to change the options as it goes. For maccel, we have the reset option; we could add a new set of call that reset a specific option (add a new call in the OnnxMlirCompiler interface, and then remove the RESET for maccel), or reset every time in the new list that you did here. I would prefer a "reset" call, it seems cleaner and then we can also remove the RESET option